plugins/net_plugin/net_plugin.cpp

```diff
@@ -315,6 +315,86 @@ namespace eosio {
    constexpr uint32_t signed_block_which       = fc::get_index<net_message, signed_block>();       // see protocol net_message
    constexpr uint32_t packed_transaction_which = fc::get_index<net_message, packed_transaction>(); // see protocol net_message
 
+   class connections_manager {
+      alignas(hardware_destructive_interference_size)
+      mutable std::shared_mutex connections_mtx;
+      std::set<connection_ptr>  connections;
+      chain::flat_set<string>   supplied_peers;
```

Comment:
why is one a `std::set` and the other a `chain::flat_set`? I'd say go flat_set all the way :-).

Reply:
Changed to `flat_set`.